openedx/features/genplus_features/genplus/models.py

```diff
@@ -114,6 +114,7 @@ def save(self, *args, **kwargs):
 class Student(models.Model):
     gen_user = models.OneToOneField(GenUser, on_delete=models.CASCADE, related_name='student')
     character = models.ForeignKey(Character, on_delete=models.SET_NULL, null=True, blank=True)
+    scn = models.CharField(unique=True, null=True, blank=True, max_length=255)
```

Comment:
Wouldn't it make sense to have SCN in GenUser?

Reply:
SCN number is only for students